openslides/mediafiles/static/templates/mediafiles/slide_mediafile.html

```diff
@@ -1,6 +1,18 @@
-<div ng-controller="SlideMediafileCtrl" class="content fullscreen">
-    <ng-pdf template-url="/static/templates/mediafiles/slide_mediafile_partial.html"
-        ng-attr-scale="{{ element.scale }}"
-        ng-attr-page="{{ element.page }}">
+<link rel="stylesheet" href="static/css/mediafiles/projector.css" />
```

Comment:
Remove this line and add all your mediafile css classed to core/static/css/projector.css (currently we use only one css file for projector, we will split it later)